analyzers/tests/SonarAnalyzer.UnitTest/ResourceTests/FilesToAnalyze/InvalidValuesFileToAnalyze.txt

```diff
@@ -0,0 +1,4 @@
+﻿foo
+123
+C:\Projects\^^^\$\Controllers\HomeController.vb
+C:\Projects/DummyProj/Views\Web.confiG
```

Comment:
Why is this line in `Invalid` file? There's already a mixed `/\` case in valid file:
```
C:\Projects/DummyProj/Views\Web.confiG
```

Reply:
I`ll rename the file. To something like FilesToAnalyzeWithInvalidValues. The idea was to check that the file is fully parsed even with invalid values.